sql/exploits.sql

```diff
@@ -0,0 +1,36 @@
+CREATE OR REPLACE FUNCTION exploit_alive () RETURNS boolean AS $$
+    plv8.elog(INFO, "Alive!");
+    return true;
+$$ language plv8;
+
+select exploit_alive();
+DO $$
+    '1'.repeat(0x7ffffff).split('');
+$$ language plv8;
+select exploit_alive();
+DO $$
+    '1'.repeat(0x7ffffff).split(/./);
+$$ language plv8;
+select exploit_alive();
+DO $$
+    '1'.repeat(0x7ffffff).split(/1/);
+$$ language plv8;
+select exploit_alive();
+DO $$
+    [].join.call('1'.repeat(0x7ffffff));
+$$ language plv8;
+select exploit_alive();
+DO $$
+    [].slice.call('1'.repeat(0x7ffffff));
+$$ language plv8;
+select exploit_alive();
+DO $$
+    [].sort.call('1'.repeat(0x7ffffff));
+$$ language plv8;
+select exploit_alive();
+DO $$
+    JSON.parse('[' + '1,'.repeat(0x7fffffd) + '1]');
+$$ language plv8;
+select exploit_alive();
+
+DROP FUNCTION exploit_alive;
```

Comment:
all of this is awesome.

Reply:
You can paste these into Chrome(ium) and see how it explodes spectacularly!

Reply:
I'd rather not :). I compiled and tried a fork bomb on a sparc 5 in the 90's, just to see for myself what it did, but in this case I'll just continue to use safari.